iron-overlay-manager.html

```diff
@@ -78,10 +78,58 @@
     },
 
     /**
+     * Brings the overlay at the specified index to the front.
+     * @param {number|Element} i
+     */
+    bringToFront: function(i) {
```

Comment:
Uhm, what if i make this private & expose it when someone actually asks for it? For what concerns this PR, we need the method `bringToFrontByIndex` (for perf reasons since we already have the index when we use it).
Anyways yes, I also agree we could be more explicit by name, or by accepting an object as input.